kornia/feature/__init__.py

```diff
@@ -10,10 +10,6 @@
 )
 from .nms import NonMaximaSuppression2d, nms2d, NonMaximaSuppression3d, nms3d
 
-# Backward compatibility
```

Comment:
is it still needed?

Reply:
I guess we could keep just one interface. /cc @ducha-aiki